packages/request-client.js/test/declarative-payments.test.ts

```diff
@@ -0,0 +1,505 @@
+import axios from 'axios';
```

Comment:
Unchanged tests, just moved outside of `index.test.ts` because it is a huge section.